### PR TITLE
VsoAgentBuildNumberTests refactored with accordance to the remark from @asbjornu

### PR DESCRIPTION
**Pull request by dpurge**
No functional changes - test refactored in accordance to the sugestion from @asbjornu

### CODE REVIEW

File: src/GitVersionCore.Tests/BuildServers/VsoAgentBuildNumberTests.cs [hunk 1]
```diff
@@ -3,65 +3,49 @@
...
 [TestFixture]
 public class VsoAgentBuildNumberTests
 {
     string key = "BUILD_BUILDNUMBER";
     string logPrefix = "##vso[build.updatebuildnumber]";
-    List<Tuple<string, string, string>> examples;
     VsoAgent versionBuilder = new VsoAgent();
 
     [SetUp]
     public void SetUpVsoAgentBuildNumberTest()
```
Comment (asbjornu) [inline src/GitVersionCore.Tests/BuildServers/VsoAgentBuildNumberTests.cs]: The SetUp method can probably be removed now, no?

**Comment by dpurge** [comment]
Empty test setup method removed.

**Comment by asbjornu** [comment]
Hm, did I forget to merge this? I've been on a holiday so I can't really remember what the story was here. Why did you close the PR, @dpurge?

**Comment by dpurge** [comment]
By mistake :) I removed the branch from my fork. I have restored the branch and I am re-opening pull request.

**Pull request merged**